packages/react/src/Alert/Alert.tsx

```diff
@@ -5,7 +5,7 @@
 
 import { AlertIcon, CheckmarkIcon, CloseIcon } from '@amsterdam/design-system-react-icons'
 import clsx from 'clsx'
-import React, { ForwardedRef, forwardRef, HTMLAttributes, PropsWithChildren } from 'react'
+import { ForwardedRef, forwardRef, HTMLAttributes, PropsWithChildren, useMemo } from 'react'
```

Comment:
Legit change but I see a trend where you make corrections in many other components then what your PR is about. I have the tendency to do the same but I usually get the order to stop it.